Mage.Sets/src/mage/cards/n/Necromentia.java

```diff
@@ -50,6 +50,7 @@ public Necromentia copy() {
     }
 }
 
+//Based on SearchTargetGraveyardHandLibraryForCardNameAndExileEffect
```

Comment:
is it possible to make an overrideable method that applies an effect based on the value? not required.

Reply:
That could work, but right now this is the only card that does anything other than drawing so I don't think it's worth the complexity.